modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -407,6 +407,7 @@ class NDB_Form_Imaging_Browser extends NDB_Form
                 );
                 if (!empty($QCExists)) {
                     $updateWhere['FileID'] = $curFileID;
+                    Utility::nullifyEmpty($updateSet, 'QCStatus');
```

Comment:
I just looked at the definition of nullifyEmpty and while this will work, we should probably use start using `$updateSet = Utility::nullifyEmpty($updateSet, 'QCStatus')` instead (and make a ticket to update the other places using nullifyEmpty) so that we can remove the side-effect from the function in the future.